Guano icecube announcement news (#1589)
nasa-gcn/gcn.nasa.gov: app/routes/news.mdx

## 2023

    <CollectionItem
      className="maxw-none"
      variantComponent={
        <CollectionCalendarDate datetime={'October 27, 2023'} />
      }
    >
      <CollectionHeading headingLevel="h3">
        <Anchor>New Swift-BAT/GUANO and IceCube Notices Available via GCN Kafka</Anchor>
      </CollectionHeading>
      <CollectionDescription>

>         The Swift-BAT/GUANO, IceCube, and GCN teams are pleased to announce the availability of new GCN notice types via the new GCN in JSON format.  JSON format notices can be streamed via [Kafka](/quickstart). [Self-service email subscription](/user/email) will be available soon.  These new notice types are *not* available via GCN Classic.

Reviewer: ```suggestion
        The Swift-BAT/GUANO, IceCube, and GCN teams are pleased to announce the availability of new GCN notice types via the new GCN in JSON format.  JSON format notices can be streamed via [Kafka](/quickstart). These new notice types are *not* available via GCN Classic.
```

We have _no_ idea when we will be able to deliver JSON alerts over email, if ever.

Author: I agree that we do not know when, but we do have to be able to do this.   Many users receive notices via email, and will expect to continue to be able to do so.

Reviewer: We should not promise features that do not have a certain delivery date.

        [**Swift-BAT/GUANO:**](/missions/swift#json-serialized-gcn-notice-types-in-gcn-kafka) gcn.notices.swift.bat.guano

>         - *Summary:* The GUANO system is a family of gamma-ray transient search pipelines that supplement the onboard triggering capabilities of Swift/BAT, significantly enhancing the sensitivity and sky area covered by the instrument.

Reviewer: As worded, it sounds like GUANO runs onboard, which it does not.

Author: @Tohuvavohu

Reviewer: How about: "GUANO is a ground-based re-analysis of data from Swift BAT, significantly enhancing the sensitivity and sky area covered as compared to Swift's onboard triggering."

Reviewer 2: remove 're'

Reviewer: ```suggestion
        - *Summary:* GUANO is a ground-based analysis of data from Swift BAT, significantly enhancing the sensitivity and sky area covered as compared to Swift's onboard triggering.
```

        - *Occurence Rate:* The Swift/BAT-GUANO notices occur at a rate of 50 times per year.

>         - *Time Delay:* The post-event processing and communication of alert is distributed to GCN within 10 minutes - 5 hours of detection, depending on data downlink method and space-to-ground communications resource availability.

Reviewer: ```suggestion
        - *Time Delay:* The post-event processing and communication of alert is distributed to GCN within 10 minutes to 5 hours of detection, depending on data downlink method and space-to-ground communications resource availability.
```

Reviewer: Also "downlink method and space-to-ground communications resource availability" is a little redundant because "downlink" and "space-to-ground communications" are synonyms. I suggest instead, "downlink method and resource availability."

        - *Notice Type:* The new [Swift/BAT-GUANO notice type](/missions/swift#json-serialized-gcn-notice-types-in-gcn-kafka) (gcn.notices.swift.bat.guano) provides timestamps, sky locations, durations, and other supplementary information on gamma-ray transients discovered through this system, to enable prompt follow-up by observers and other instruments.  The 'initial' notice distributed for any detection will not include localization information. In the majority of cases later 'update' notices will follow with a localization attached. ~1/4 of GUANO bursts will have arcminute-scale localizations, distributed in the form [RA, Dec, Error] the remainder will be more poorly localized and may be distributed as HEALPix maps.

        [**IceCube LVK Neutrino Track Search:**](/missions/icecube#gcn-notice-types-in-gcn-over-kafka) gcn.notices.icecube.lvk_nu_track_search

>         - *Summary:* IceCube is a is a cubic-kilometer Cherenkov particle detector deployed in the Antarctic ice beneath the Amundsen-Scott South Pole Station. IceCube detects neutrinos by observing the light produced by relativistic charged particles produced by neutrino interactions in or near the instrumented volume of ice.

Reviewer: Vary word choice to decrease redundancy:

```suggestion
        - *Summary:* IceCube is a is a cubic-kilometer Cherenkov particle detector deployed in the Antarctic ice beneath the Amundsen-Scott South Pole Station. IceCube detects neutrinos by observing the light produced by relativistic charged particles created by neutrino interactions in or near the instrumented volume of ice.
```

>         - *Occurrence Rate:* The LVK Nu Track Search notices are triggered by LVK alerts (missions/lvk) and occur at the same rate of these alerts, searching for neutrino signals from both significant and low significance LVK alerts.

Reviewer: ```suggestion
        - *Occurrence Rate:* The LVK Nu Track Search notices are triggered by [LIGO/Virgo/Kagra alerts](/missions/lvk) and occur at the same rate of these alerts, searching for neutrino signals from both significant and low significance LVK alerts.
```
